src/MultiMessageSender.sol

```diff
@@ -133,104 +148,165 @@ contract MultiMessageSender {
     /// @param _nativeValue is the value to be sent to _target by low-level call (eg. address(_target).call{value: _nativeValue}(_callData))
     /// @param _expiration refers to the number of days that a message remains valid before it is considered stale and can no longer be executed.
     /// @param _refundAddress refers to the refund address for any extra native tokens paid
+    /// @param _fees refers to the fees to pay to each adapter
```

Comment:
nit: comment on line 144 is stale, since `estimteTotalMessageFee()` has been removed

Reply:
Updated.